api/scpca_portal/test/test_loader.py

```diff
@@ -0,0 +1,311 @@
+        self.assertEqual(project.downloadable_sample_count, 0)
+        self.assertTrue(project.has_single_cell_data)
+        self.assertTrue(project.has_spatial_data)
+        self.assertEqual(project.human_readable_pi_name, "TBD")
+        self.assertTrue(project.includes_anndata)
+        self.assertFalse(project.includes_cell_lines)
+        self.assertTrue(project.includes_merged_sce)
+        self.assertTrue(project.includes_merged_anndata)
+        self.assertFalse(project.includes_xenografts)
+
+        self.assertIn(Sample.Modalities.NAME_MAPPING["BULK_RNA_SEQ"], project.modalities)
+        self.assertIn(Sample.Modalities.NAME_MAPPING["SPATIAL"], project.modalities)
+        self.assertEqual(project.multiplexed_sample_count, 0)
+        self.assertIn("Homo sapiens", project.organisms)
+        self.assertEqual(project.pi_name, "scpca")
+        self.assertEqual(project.s3_input_bucket, settings.AWS_S3_INPUT_BUCKET_NAME)
+
+        # single_cell samples: SCPCS999990, SCPCS999997
+        # spatial samples: SCPCS999991
+        # single_cell sample SCPCS999994 is unavailable
+        # As such, there should be 3 samples, but 4 are processed
+        # Like we do for projects where we check if a dir exists with data before processing,
+        # we should do the same for samples
+        # The check should be as follows
+        # single_cell, spatial = 2, 1
+        # self.assertEqual(project.sample_count, single_cell + spatial)
+
+        self.assertIn("cell", project.seq_units)
+        self.assertIn("spot", project.seq_units)
+        self.assertIn("10Xv3", project.technologies)
+        self.assertIn("visium", project.technologies)
+        self.assertEqual(project.title, "TBD")
+
+        # single_cell sample SCPCS999994 is unavailable
+        # The following evaluates to 0, whereas it should be one
+        # self.assertEqual(project.unavailable_samples_count, 1)
+
+        # CHECK SAMPLE VALUES
+        sample0 = project.samples.filter(scpca_id="SCPCS999990").first()
+        self.assertEqual(sample0.age, "2")
+        self.assertEqual(sample0.age_timing, "diagnosis")
+        self.assertIsNone(sample0.demux_cell_count_estimate)
+        self.assertEqual(sample0.diagnosis, "diagnosis1")
+        self.assertEqual(sample0.disease_timing, "Initial diagnosis")
+        self.assertFalse(sample0.has_multiplexed_data)
+        self.assertTrue(sample0.has_single_cell_data)
+        self.assertFalse(sample0.has_spatial_data)
+        self.assertTrue(sample0.includes_anndata)
+        self.assertFalse(sample0.is_cell_line)
+        self.assertFalse(sample0.is_xenograft)
+        self.assertListEqual(sample0.multiplexed_with, [])
+        self.assertEqual(sample0.sample_cell_count_estimate, 3432)
+        self.assertEqual(sample0.seq_units, "cell")
+        self.assertEqual(sample0.technologies, "10Xv3")
+        self.assertEqual(sample0.tissue_location, "tissue1")
+        self.assertEqual(sample0.treatment, "")
```

Comment:
Make an `assertSample` helper function that takes in a dictionary with all of the correct values.